pygmt/tests/test_plot3d.py

```diff
@@ -517,3 +518,57 @@ def test_plot3d_deprecate_columns_to_incols(data, region):
         )
         assert len(record) == 1  # check that only one warning was raised
     return fig
+
+
+@pytest.mark.mpl_image_compare
+def test_plot3d_gmt_shapfile_MultiPoint_default_style():
```

Comment:
pylint doesn't like this name style, use snake_case name style instead.
```suggestion
def test_plot3d_gmt_shapfile_multipoint_default_style():
```

Reply:
Maybe rename it to `test_plot3d_ogrgmt_file_multipoint_default_style` since the `.gmt` file isn't really a shapefile? Will need to rename the `*.png.dvc` and `*.dvc` files too.

Reply:
> Maybe rename it to test_plot3d_ogrgmt_file_multipoint_default_style since the .gmt file isn't really a shapefile? Will need to rename the *.png.dvc and *.dvc files too.

I changed the names as so.